src/index.ts

```diff
@@ -23,6 +23,12 @@ client.on("message", (message) => {
         //Are they on sale?
         pubSub(message);
         break;
+      case "image":
+        imageSearch(message);
+        break;
+      case "animate":
+        imageSearch(message);
+        break;
```

Comment:
Can you do 
```
case "image":
case "animate":
  imageSearch(message);
  break;
```
Since the cases for both are identical?  For both this one and the "dm" case